sci-libs/calculix-ccx/files/01_ccx_2.8p2_Makefile_custom_cc_flags_spooles_arpack.patch

```diff
@@ -0,0 +1,31 @@
+diff -ru CalculiX.orig/ccx_2.8p2/src/Makefile CalculiX/ccx_2.8p2/src/Makefile
```

Comment:
Please clean up the patch and add a short description like explained in http://dev.gentoo.org/~vapier/clean-patches